### ISSUE TITLE
Error rendering LineChart with zero-only data

### ISSUE THREAD

**Issue by micalm** [indiespirit/react-native-chart-kit#15]
Hi,
I've encountered a problem trying to render a following chart:

```javascript
<LineChart
    data={{
        labels: ['January', 'February', 'March', 'April', 'May', 'June'],
        datasets: [{
            data: [0, 0, 0, 0, 0, 0]
        }]
    }}
    width={Dimensions.get('window').width} // from react-native
    height={220}
    chartConfig={{
        backgroundColor: '#2c3e50',
        backgroundGradientFrom: '#2e5a80',
        backgroundGradientTo: '#2e5a85',
        color: (opacity = 1) => `rgba(255, 255, 255, ${opacity})`,
        style: {
            borderRadius: 16
        }
    }}
    style={{
        marginVertical: 8,
    }}
/>
```

After changing the data array to anything that doesn't have only zeroes (eg. [1,0,0,0,0,0]) in it, the chart renders properly. BarChart renders this exact same attributes fine. The error I'm getting:

```
Error while updating property 'd' in shadow node of type: RNSVGPath

null

Invalid float: "N"
updateShadowNodeProp
    ViewManagersPropertyCache.java:113
setProperty
    ViewManagerPropertyUpdater.java:154
updateProps
    ViewManagerPropertyUpdater.java:58
updateProperties
    ReactShadowNodeImpl.java:378
createView
    UIImplementation.java:289
createView
    UIManagerModule.java:366
invoke
    Method.java
invoke
    JavaMethodWrapper.java:372
invoke
    JavaModuleWrapper.java:160
run
    NativeRunnable.java
handleCallback
    Handler.java:739
dispatchMessage
    Handler.java:95
dispatchMessage
    MessageQueueThreadHandler.java:29
loop
    Looper.java:148
run
    MessageQueueThreadImpl.java:192
run
    Thread.java:818

```

Package versions:
**react** 16.3.1
**react-native** 0.55.4
**react-native-chart-kit** 0.1.5
**react-native-svg** 5.5.1
**paths-js** 0.4.7

Tested on both emulated && physical Android 7.0

Any help would be appreciated.

**Comment by micalm** [comment]
Update:
The error occurs when all data points are equal, so for example `[3000,3000,3000,3000,3000,3000]`  does not work too.

**Comment by foufee** [comment]
Unfortunately this basically is a show stopper for using this, any idea on where it might be?

**Comment by Hermanya** [comment]
Fixed in 1.1.0

**Comment by Daniyal-Habib-Tez** [comment]
Still @Hermanya gets error when dataset consist of [0,0,0,0,0,0]
